src/controller/reviews.controller.js

```diff
@@ -0,0 +1,118 @@
+import Review from "../model/review.model.js";
+import { response } from "express";
+
+export const reviewsGet = async (req, res = response) => {
+  const { limite, desde } = req.query;
+  const query = {};
+
+  const [total, reviews] = await Promise.all([
+    Review.countDocuments(query),
+    Review.find(query).skip(Number(desde)).limit(Number(limite)),
+  ]);
```

Comment:
mismo problema que el anterior, cambiar param `desde` a `page` y no es necesario declarar `query` porque no estamos filtrando por nada. Recuerda formatear el codigo acorde
```suggestion
export const reviewsGet = async (req, res = response) => {
  const { limite, page } = req.query;

  const [total, reviews] = await Promise.all([
    Review.countDocuments(),
    Review.find()
      .skip(Number(page) * Number(limit))
      .limit(Number(limite)),
  ]);
```